template-haskell-optics/CHANGELOG.md

```diff
@@ -1,2 +1,5 @@
+# template-haskell-optics-0.2
+* TBW: No DataPrism
```

Comment:
`DataPrism` wasn't exported, so it probably doesn't matter.

Reply:
I defer completion and correction of that changelog to you for later.